src/pages/guidelines/color/overview.mdx

```diff
@@ -261,54 +289,230 @@ attention or create visual tension.
   </DoDont>
 </DoDontRow>
 
+<br />
+
+### Tokens
+
+Color in Carbon is managed through tokens. Tokens are an abstracted method of
```

Comment:
Maybe managed -> "organized with tokens", or "is applied to components with tokens"

```suggestion
Color in Carbon is managed through tokens. Tokens are an abstracted method of
```